common/lib/xmodule/xmodule/modulestore/xml_importer.py

```diff
@@ -30,26 +30,26 @@
 import re
 from lxml import etree
 
-from xblock.fields import XBlockMixin
-from xmodule.library_tools import LibraryToolsService
-from xmodule.modulestore.xml import XMLModuleStore, LibraryXMLModuleStore, ImportSystem
-from xblock.runtime import KvsFieldData, DictKeyValueStore
-from xmodule.x_module import XModuleDescriptor, XModuleMixin
 from opaque_keys.edx.keys import UsageKey
-from xblock.fields import Scope, Reference, ReferenceList, ReferenceValueDict
-from xmodule.contentstore.content import StaticContent
-from .inheritance import own_metadata
-from xmodule.errortracker import make_error_tracker
-from .store_utilities import rewrite_nonportable_content_links
 import xblock
+from xblock.core import XBlockMixin
+from xblock.fields import Scope, Reference, ReferenceList, ReferenceValueDict
+from xblock.runtime import KvsFieldData, DictKeyValueStore
```

Comment:
Does it make sense to go ahead and alphabetize the imports within each line, since git is treating the lines as changed anyway?

Reply:
It does - I let `isort` do the work for me. 😄